### ISSUE TITLE
Please explain the loss fuction of ssd. ^^

### ISSUE THREAD

**Issue by arurumang** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#415]
I've been using ssd for learning and I'm interested in seeing different loss fuctions for fast rcnn and ssd. Please tell me why.
